src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -2006,11 +1885,11 @@ MavlinkReceiver::handle_message_hil_sensor(mavlink_message_t *msg)
 	mavlink_hil_sensor_t imu;
 	mavlink_msg_hil_sensor_decode(msg, &imu);
 
-	uint64_t timestamp = hrt_absolute_time();
+	const uint64_t timestamp = hrt_absolute_time();
```

Comment:
Hi @dagar, since this line is changing anyway, you could update this from `uint64_t` to `hrt_abstime`

Reply:
Done.